### DISCUSSION TITLE
no mention of draid in man pages of debian packages

### DISCUSSION THREAD

**Discussion by AllKind** [openzfs/zfs#11732] [Q&A] [answered]
Good day,

yesterday I installed zfs 2.0.4 from https://launchpad.net/~jonathonf/+archive/ubuntu/zfs for ubuntu bionic (18.04).

When reading the man pages zpoolconceps and zpool-create I noticed there was no mentioning of draid.
I looked at master branch of openzfs on github and these man pages contain information about draid.
I took a look at zfs-linux_2.0.4.orig.tar.gz from the ppa and at the debian package site. No word about draid I could find.
I took a brief look at the Makefile.* of the debian package in /man, because I thought there might be some automagic to detect if I upgraded my pool to the newest feature level and then automagically upgrade the man pages, but couldn't see anything pointing to that.
Can anyone confirm my observation of missing draid documentation?
Maybe there's a bug in the openzfs build system, or upstream at debian?

I'm happy to report it upstream, but would like to first make sure, by asking for confirmation here from more experienced people than me.

Thank you!

**Comment by gmelikov** [comment, marked as answer]
Draid isn't included in 2.0 release, it's only in master now.

**Reply by AllKind** [reply]
Ah, ok... my fault then :)
